FirebaseAI/Tests/TestApp/Tests/Utilities/InstanceConfig.swift

```diff
@@ -107,11 +107,11 @@ struct InstanceConfig {
 extension InstanceConfig: CustomTestStringConvertible {
   var testDescription: String {
     let endpointSuffix = switch apiConfig.service.endpoint {
-    case .firebaseVertexAIProd:
+    case .firebaseProxyProd:
       ""
-    case .firebaseVertexAIStaging:
+    case .firebaseProxyStaging:
       " - Staging"
-    case .generativeLanguage:
+    case .geminiDeveloperDirect:
       " - Generative Language"
```

Comment:
Maybe " - Generative Language (No proxy)" To be really clear for future us.

Reply:
Good idea. Updated this to "Google AI" and "(Bypass Proxy)".